src/pages/assets.tsx

```diff
@@ -74,7 +74,34 @@ const Header = styled.header`
   }
 `
 
-const AssetsPage = ({ data }) => {
+export interface IGatsbyChildImageSharp {
+  gatsbyImageData: {
+    height: number
+    width: number
+    images: {
+      fallback: {
+        src: string
+        srcSet: string
+        sizes: string
+      }
+      sources: {
+        srcSet: string
+        sizes: string
+        type: string
+      }[]
+    }
+    layout: string
+    placeholder: {
+      fallback: string
+    }
+  }
+}
+
+export interface IProps {
+  data: Record<string, IGatsbyChildImageSharp>
+}
+
+const AssetsPage: React.FC<IProps> = ({ data }) => {
```

Comment:
As mentioned in the other PR, lets use `PageProps` from gatsby and use the auto-generated types using `Queries.{NameOfTheQuery}`.